src/darsia/utils/linear_solvers/solver.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional, Union
+
+import numpy as np
+
+
+class Solver:
```

Comment:
All names (Solver, CG, MG, Jacobi) sound very generic though have a very tailored use case in mind. OK for now. We will see whether we want to change names at some point.

Reply:
I thought about it, but I thought that should be for when we actually want to solve other linear systems. As of now I have not come up with any.